src/main/webapp/clusters-view/clusters-view.controller.js

```diff
@@ -12,8 +12,77 @@ angular.module('managementConsole')
             $scope.group = modelController.getServer().getServerGroupName();
             $scope.stack = modelController.getServer().getFirstServer().getDefaultStack();
             $scope.relays = {};
+
             //endpoints for each cache container
             $scope.endpoints = [];
+
+            $scope.offlineSites = {};
+            $scope.onlineSites  = {};
+            $scope.mixedSites   = {};
+
+            //
+            // Updates the map of remote site status
+            //
+            $scope.refreshRemoteSitesStatus = function(cluster) {
+              var resourcePathCacheContainer = cluster.domain.getFirstServer().getResourcePath()
```

Comment:
@tristantarrant told me we'll probably have to replace this call with domain.getCoordinator() eventually